html-css/examples/css/chapter15.css

```diff
@@ -0,0 +1,195 @@
+
+/* NORMAL FLOW */
+.normal-flow {
+    width: 750px;
+    font-family: Arial, Verdana, sans-serif;
+    color: #665544;
+}
+
+.normal-flow > h1 {
+    background-color: #efefef;
+    padding: 10px;
+}
+
+.normal-flow > p {
+    width: 450px;
+}
+
+
+/* RELATIVE POSITIONING */
+.relative-positioning > .example {
+    position: relative;
+    top: 10px;
+    left: 100px;
+}
+
+
+
+/* ABSOLUTE POSITIONING */
+.absolute > h1 {
+    position: absolute;
+    top: 0px;
+    left: 500px;
+    width: 250px;
+}
+
+.absolute > p {
+    width: 450px;
+}
+
+
+/* FIXED POSITIONING */
+.fixed > h1 {
+    position: fixed;
+    top: 0px;
+    left: 50px;
+    padding: 10px;
+    margin: 0px;
+    width: 100%;
+    background-color: #efefef;
+}
+
+.fixed > .example {
+    margin-top: 100px;
+}
+
+
+/* OVERLAPPING ELEMENTS */
+.overlapping > h1 {
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    margin: 0px;
+    padding: 10px;
+    width: 100%;
+    background-color: #efefef;
+    z-index: 10;
+}
+
+.overlapping > p {
+    position: relative;
+    top: 70px;
+    left: 70px;
+}
+
+
+/* FLOATING ELEMENTS */
+blockquote {
+    float: right;
+    width: 275px;
+    font-size: 130%;
+    font-style: italic;
+    font-family: Georgia, Times, serif;
+    margin: 0px 0px 10px 10px;
+    padding: 10px;
+    border-top: 1px solid #665544;
+    border-bottom: 1px solid #665544;
+}
+
+
+
+/* USING FLOAT TO PLACE ELEMENTS SIDE-BY-SIDE */
+.use-float {
+    width: 750px;
+    font-family: Arial, Verdana, sans-serif;
+    color: #665544;
+}
+
+.use-float > p {
+    width: 230px;
+    float: left;
+    margin: 5px;
+    padding: 5px;
+    background-color: #efefef;
+}
+
+
+/* CLEARING FLOATS */
+.clearing {
+    width: 750px;
+    font-family: Arial, Verdana, sans-serif;
+    color: #665544;
+}
+
+.clearing > p {
+    width: 230px;
+    float: left;
+    margin: 5px;
+    padding: 5px;
+    background-color: #efefef;
+}
+
+.clearing > .clear {
+    clear: left;
+}
+
+
+
+/* PARENTS OF FLOATED ELEMENTS: PROBLEM */
+.problem {
+    border: 1px solid #665544;
+}
+
+
+/* PARENTS OF FLOATED ELEMENTS: SOLUTION */
+div {
+    border: 1px solid #665544;
+    overflow: auto;
+    width: 100%;
+}
+
+
+/* CREATING MULTI-COLUMN LAYOUTS WITH FLOATS */
+.column1of2 {
+    float: left;
+    width: 620px;
+    margin: 10px;
+}
+
+.column2of2 {
+    float: left;
+    width: 300px;
+    margin: 10px;
+}
+
+
+/* A FIXED LAYOUT */
+.fixed {
+    width: 960px;
+    margin: 0 auto;
+}
+
+.fixed>#content {
+    overflow: auto;
+    height: 100%;
+}
+
+.fixed> #header>#nav, .fixed>#content#feature, .fixed>#footer {
```

Comment:
should be `.fixed > #header > #nav`, please read css style guide

Reply:
Thank you for your comments. I will learn from experience